draft-ietf-quic-tls.md

```diff
@@ -659,9 +659,9 @@ exposing TLS record protection keys.
 
 ### Initial Secrets {#initial-secrets}
 
-Packets that carry the TLS handshake (Initial, Retry, and Handshake) are
-protected with a secret derived from the Destination Connection ID field from
-the client's Initial packet.  Specifically:
+Packets that carry the TLS handshake (Initial and Handshake) are protected with
+a secret derived from the Destination Connection ID field from the client's
```

Comment:
maybe say this is called the Initial secret?

Reply:
Done, sort of.